README.md

```diff
@@ -92,14 +93,14 @@ OPTIONS
   --id=id                        API Key unique ID
   --password=password            Kuzzle user password
   --ssl                          Use SSL to connect to Kuzzle
-  --username=username            [default: anonymous] Kuzzle user
+  --username=username            [default: anonymous] Kuzzle username (local strategy)
 ```
 
-_See code: [src/commands/api-key/create.ts](https://github.com/kuzzleio/kourou/blob/v0.6.0/src/commands/api-key/create.ts)_
+_See code: [src/commands/api-key/create.ts](https://github.com/kuzzleio/kourou/blob/v0.8.0/src/commands/api-key/create.ts)_
 
 ## `kourou api-key:delete USER`
 
-Deletes a user's API Key.
+Deletes a new API Key for an user
```

Comment:
The previous form was correct, you cannot delete an API key if it's new, and it's "a user":

```suggestion
Deletes a user's API Key.
```